docs/user-guide/src/bmo/firmware_components.md

```diff
@@ -0,0 +1,159 @@
+# Firmware Components
+
+Metal3 supports modifying firmware components of the bare metal hosts.
```

Comment:
In a similar vain, "modifying firmware components" might be a confusing way to put it for unprepared users. Let's expand. "supports updating firmware and retrieving the current firmware versions" or something like this.